common/changes/@microsoft.azure/openapi-validator-rulesets/akhilailla-implement_rpc_uri_v1_v12_2023-09-13-23-08.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "implement rpc-uri-v1-12",
```

Comment:
>implement rpc-uri-v1-12 [](http://example.com/codeflow?start=18&length=23)

change log needs to be user facing. pl take a look at the guidelines posted by Brett earlier